scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerService.java

```diff
@@ -154,16 +154,20 @@ boolean restartInErrorTask(String sessionId, Integer jobId, String taskName)
      */
     String getTasks(String sessionId, String jobId, int offset, int limit) throws RestServerException, ServiceException;
 
+    String getTasks(String sessionId, String jobId, int offset, int limit, String statusFilter)
+            throws RestServerException, ServiceException;
+
     /**
      * Gets the list of tasks that correspond to a job and filtered by a given tag.
      * @param sessionId the session if of the user that asks for the tasks
      * @param jobId the id of the job for which its task list is asked 
-     * @param tag the tag used to filter the tasks.
      * @param offset the first task to show
      * @param limit the last task to show
+     * @param tag the tag used to filter the tasks.
+     * @param statusFilter aggregation status to apply in filter
      * @return the list of tasks as raw json
      */
-    String getTasksByTag(String sessionId, String jobId, String tag, int offset, int limit)
+    String getTasksByTagByStatus(String sessionId, String jobId, int offset, int limit, String tag, String statusFilter)
```

Comment:
I'm not sure the name of this function is very clear. Is by tag AND by status? Or by tag OR by status?